orc8r/cloud/go/services/certifier/storage/storage_blobstore_test.go

```diff
@@ -413,16 +439,42 @@ func TestCertifierBlobstore_DeleteCertInfo(t *testing.T) {
 func TestCertifierBlobstore_ListSerialNumbers(t *testing.T) {
 	var blobFactMock *mocks.BlobStorageFactory
 	var blobStoreMock *mocks.TransactionalBlobStorage
+
 	someErr := errors.New("generic error")
+
+	info0 := &protos.CertificateInfo{
+		Id:        nil,
+		NotBefore: &timestamp.Timestamp{Seconds: 0xdead, Nanos: 0xbeef},
+		NotAfter:  &timestamp.Timestamp{Seconds: 0xbeef, Nanos: 0xfeed},
+		CertType:  1,
+	}
+	info1 := &protos.CertificateInfo{
+		Id:        nil,
+		NotBefore: &timestamp.Timestamp{Seconds: 0xaaaa, Nanos: 0xbbbb},
+		NotAfter:  &timestamp.Timestamp{Seconds: 0xcccc, Nanos: 0xdddd},
+		CertType:  2,
+	}
+	marshaledInfo0, err := proto.Marshal(info0)
+	assert.NoError(t, err)
+	marshaledInfo1, err := proto.Marshal(info1)
+	assert.NoError(t, err)
+
 	serialNumbers := []string{"serial_number_0", "serial_number_1"}
```

Comment:
Invert this -- create `serialNumbers` after creating `blobs` as `serialNumbers := blobs.Keys()`, then check that `serialNumbers` is equal to `"[]string{"serial_numbrer_0", ...}`